src/WebJobs.Script/Binding/FunctionBinding.cs

```diff
@@ -250,6 +230,18 @@ public static void ConvertValueToStream(object value, Stream stream)
                 {
                     bytes = Encoding.UTF8.GetBytes((string)value);
                 }
+                else if (type == typeof(int))
```

Comment:
In my test I was writing a single int to the blob which was failing because we were missing 3 of the Node types.